lib/ex_doc/formatter/epub/templates.ex

```diff
@@ -68,4 +70,16 @@ defmodule ExDoc.Formatter.EPUB.Templates do
   EEx.function_from_file(:defp, :head_template,
                          Path.expand("templates/head_template.eex", __DIR__),
                          [:config, :page])
+
+  @media_types
+  |> String.split("\n", trim: true)
+  |> Enum.each(fn(line) ->
+    [extension, media] = line |> String.strip() |> String.split(",")
+
+    def media_type("." <> unquote(extension)) do
+      unquote(media)
+    end
+  end)
+
+  def media_type(_), do: nil
```

Comment:
Should we default to "application/octet-stream" or something of sorts? Otherwise the media-type field will be empty, or is that indeed the preferred behaviour?